### ISSUE TITLE
feature / question : runorama exists?

### ISSUE THREAD

**Issue by nphias** [holochain/tryorama#27]
Hi there,
just wondering if it would be possible to adapt tryorama to run against a real conductor.. not one that is spawned and killed.. It would greatly assist with testing and data import

as anything like that been discussed?

**Comment by maackle** [comment]
@nphias most of the work tryorama does is around config generation and environment management, i.e. safely spawning and killing multiple conductors and keeping them from stepping on each others' toes.

It sounds like you just want to run a script that interacts with a conductor that you already configured and are running, which you can totally do without tryorama, by just using [hc-web-client](https://github.com/holochain/hc-web-client) to connect to a conductor and call zome functions and such. It would be missing `await s.consistency()`, but I think that's about it. If you wanted that functionality too, I think it wouldn't be too hard to pull in [hachiko](https://github.com/holochain/hachiko) separately to do the consistency awaiting.

Basically tryorama's main purpose is to do exactly the stuff you don't need in this scenario.

Would that accomplish what you want?

**Comment by nphias** [comment]
yeah i dont really want to re-create what i am doing in tryorama.. in some other javascript file.. (command line).. . i think this might be possible with TryCP somehow.

**Comment by maackle** [comment]
Maybe so, but TryCP is just about spawning and killing conductors on machines other than the one you're running the test on. Not so much about connecting to long-lived conductors

What you're doing sounds a lot more like automated UI interactions via a script than something that a testing framework would handle. I hear you about wanting to be able to use the same syntax in tryorama for this though. That would require pulling out the Player interface and using it in a script. It shouldn't be too hard, and would be a pretty useful thing if you wanted to take a stab at it. I could help with advice, but otherwise don't have the bandwidth to make that happen right now.

**Comment by jost-s** [comment]
@nphias This looks fairly stale - is this still a request?

**Comment by nphias** [comment]
mmm.  yeah i think that request was around the need to do bulk operations (dna migration etc.) via the command line and not the websocket..  but its not something i need now and probably not so related to tryorama

**Comment by jost-s** [comment]
@nphias Right, if it's a command line action, you can formulate the request in the holochain repository. Once it becomes something you need =)
